package.json

```diff
@@ -17,6 +17,14 @@
       "types": "./dist/index.d.ts",
       "default": "./dist/index.js"
     },
+    "./types": {
+      "require": {
+        "types": "./dist/types/Pattern.d.cts"
+      },
+      "import": {
+        "types": "./dist/types/Pattern.d.ts"
+      }
+    },
```

Comment:
Could you create a `types/index.ts` file that exports the types we need from `Pattern.ts`? This should make it easier to export type from other files if necessary in the future